src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/StartupProjectRegistrar.cs

```diff
@@ -132,7 +133,8 @@ protected override Tasks.Task DisposeCoreAsync(bool initialized)
         {
             foreach (var provider in _launchProviders.Value.Debuggers)
             {
-                if (await provider.Value.CanLaunchAsync(DebugLaunchOptions.DesignTimeExpressionEvaluation)
+                if (string.Equals(DebuggerName, provider.Metadata.DebuggerName, StringComparison.OrdinalIgnoreCase) &&
+                    await provider.Value.CanLaunchAsync(DebugLaunchOptions.DesignTimeExpressionEvaluation)
```

Comment:
Why is this "DesignTimeExpressionEvaluation"? these options should be what ever options are passed when I press F5.